cli/js/web/console.ts

```diff
@@ -360,77 +384,87 @@ function createArrayString(
       } else {
-        return stringifyWithQuotes(val, ctx, level, maxLevel);
+        return stringifyWithQuotes(val, ctx, level, inspectOptions);
       }
     },
-    group: true,
+    group: !inspectOptions.alwaysWrap,
+    sort: false,
   };
-  return createIterableString(value, ctx, level, maxLevel, printConfig);
+  return createIterableString(value, ctx, level, printConfig, inspectOptions);
 }
 
 function createTypedArrayString(
   typedArrayName: string,
   value: TypedArray,
   ctx: ConsoleContext,
   level: number,
-  maxLevel: number
+  inspectOptions: Required<InspectOptions>
 ): string {
   const valueLength = value.length;
   const printConfig: IterablePrintConfig<unknown> = {
     typeName: typedArrayName,
     displayName: `${typedArrayName}(${valueLength})`,
     delims: ["[", "]"],
-    entryHandler: (entry, ctx, level, maxLevel): string => {
+    entryHandler: (entry, ctx, level, inspectOptions): string => {
       const [_, val] = entry;
-      return stringifyWithQuotes(val, ctx, level + 1, maxLevel);
+      return stringifyWithQuotes(val, ctx, level + 1, inspectOptions);
     },
-    group: true,
+    group: !inspectOptions.alwaysWrap,
+    sort: false,
   };
-  return createIterableString(value, ctx, level, maxLevel, printConfig);
+  return createIterableString(value, ctx, level, printConfig, inspectOptions);
 }
 
 function createSetString(
   value: Set<unknown>,
   ctx: ConsoleContext,
   level: number,
-  maxLevel: number
+  inspectOptions: Required<InspectOptions>
 ): string {
   const printConfig: IterablePrintConfig<unknown> = {
     typeName: "Set",
     displayName: "Set",
     delims: ["{", "}"],
-    entryHandler: (entry, ctx, level, maxLevel): string => {
+    entryHandler: (entry, ctx, level, inspectOptions): string => {
       const [_, val] = entry;
-      return stringifyWithQuotes(val, ctx, level + 1, maxLevel);
+      return stringifyWithQuotes(val, ctx, level + 1, inspectOptions);
     },
     group: false,
+    sort: inspectOptions.sortKeys,
   };
-  return createIterableString(value, ctx, level, maxLevel, printConfig);
+  return createIterableString(value, ctx, level, printConfig, inspectOptions);
 }
 
 function createMapString(
   value: Map<unknown, unknown>,
   ctx: ConsoleContext,
   level: number,
-  maxLevel: number
+  inspectOptions: Required<InspectOptions>
 ): string {
   const printConfig: IterablePrintConfig<[unknown]> = {
     typeName: "Map",
     displayName: "Map",
     delims: ["{", "}"],
-    entryHandler: (entry, ctx, level, maxLevel): string => {
+    entryHandler: (entry, ctx, level, inspectOptions): string => {
       const [key, val] = entry;
       return `${stringifyWithQuotes(
         key,
         ctx,
         level + 1,
-        maxLevel
-      )} => ${stringifyWithQuotes(val, ctx, level + 1, maxLevel)}`;
+        inspectOptions
+      )} => ${stringifyWithQuotes(val, ctx, level + 1, inspectOptions)}`;
     },
     group: false,
```

Comment:
Why is it not `group: !inspectOptions.alwaysWrap` ?

Reply:
`group: !inspectOptions.alwaysWrap` replaces `group: true`.

Reply:
What do you mean?

Reply:
I didn't change this code. Where grouping is on, I override it depending on `alwaysWrap`. Where grouping is off, it doesn't interfere with `alwaysWrap` so I leave it alone.

Reply:
I see. Anyway I think there’s two reason for this:
1. The previous author thinks that entries of Set doesn’t need to be grouped 
2. It’s a mistake made by the previous author 

(1) seems like the more probable reason, so I think it’s better to leave a comment why the group option should be always false instead of depending on inspectOptions.wrap, else it might be confusing for future contributors.